### ISSUE TITLE
isn't it against GPL or whatever for not releasing the source code?

### ISSUE THREAD

**Issue by komarutac** [MiCode/Xiaomi_Kernel_OpenSource#27845]

**Comment by ayushqui** [comment]
Yes indeed

**Comment by ghost** [comment]
Yeah, we can actually go to https://gpl-violations.org and report them.

**Comment by ayushqui** [comment]
How to report there but .

**Comment by ayushqui** [comment]
Here mail here :- info@gpl-violations.org

**Comment by ghost** [comment]
![image](https://github.com/user-attachments/assets/d1f2ad2e-ab38-4359-a2aa-e5143a754f5a)

**Comment by ghost** [comment]
If she doesn't do it, I will be emailing https://gpl-violations.org

**Comment by ayushvps** [comment]
Yes go for it

**Comment by ghost** [comment]
> Yes go for it

She told me to submit feedback asking Xiaomi to release it, and also said most devices got their kernel source released because of it, I will wait until that feedback gets reviewed.

**Comment by ayushvps** [comment]
Can u provide ss

**Comment by komarutac** [comment]
hi, anyone alive?

**Comment by hilmidp** [comment]
so how did it go guys?

**Comment by ghost** [comment]
> so how did it go guys?

still no progress on getting source code, prebuilt is out tho
